tasks/sanity-check.yml

```
@@ -18,6 +18,13 @@
     - "{{ jenkins_user }}"
     - "{{ jenkins_version }}"
 
+- name: Ensure that startup timeout variables are valid
+  ansible.builtin.assert:
+    that:
+      - jenkins_startup_timeout_sec > 0
+      - jenkins_startup_retry_delay_sec > 0
```

I really like the idea of this assertion 👍 

Is it possible to do something like this

```suggestion
      - jenkins_startup_retry_delay_sec > jenkins_startup_timeout_sec
```

This would cover the case that it's greater than zero and also that the retry delay is greater than the startup timeout.